### ISSUE TITLE
Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined.

### ISSUE THREAD

**Issue by mayhizi** [DoubleSymmetry/react-native-track-player#1340]
**Describe the bug**

 [see for more details](https://developer.android.com/guide/topics/manifest/activity-element#exported)

**To Reproduce**

- It happens when we try to build the app.

**Environment (please complete the following information):**
System:
    OS: macOS 11.6
    CPU: (4) x64 Intel(R) Core(TM) i5-8210Y CPU @ 1.60GHz
    Memory: 92.76 MB / 8.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 14.18.1 - ~/.nvm/versions/node/v14.18.1/bin/node
    Yarn: 1.22.17 - ~/.nvm/versions/node/v14.18.1/bin/yarn
    npm: 6.14.15 - ~/.nvm/versions/node/v14.18.1/bin/npm
    Watchman: 2021.10.18.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.11.2 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 15.0, DriverKit 20.4, macOS 11.3, tvOS 15.0, watchOS 8.0
    Android SDK:
      API Levels: 29, 30, 31
      Build Tools: 30.0.2, 31.0.0
      System Images: android-29 | Google Play Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: 2020.3 AI-203.7717.56.2031.7784292
    Xcode: 13.0/13A233 - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_292 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: ^17.0.2 => 17.0.2 
    react-native: ^0.66.0 => 0.66.1 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
What `react-native-track-player` version are you using?  2.1.2
Testing on a real device 
**Code**
```sh
<receiver android:name="androidx.media.session.MediaButtonReceiver">
            <intent-filter>
                <action android:name="android.intent.action.MEDIA_BUTTON" />
            </intent-filter>
 </receiver>
```

**Comment by espenjanson** [comment]
Yup, same here!

**Comment by justinhandley** [comment]
I also am getting this after needing to upgrade android for other reasons.

**Comment by ridik1803** [comment]
Same here!

**Comment by mrigo** [comment]
Yes same here!

**Comment by juancarlosqr** [comment]
Yes, confirmed!

**Comment by annaostapenko22** [comment]
I am facing the same issue!

**Comment by lelukas** [comment]
And indeed is missing 
![image](https://user-images.githubusercontent.com/14317935/145460493-0654711e-d54f-472d-b451-b363730d710f.png)

**Comment by dcvz** [comment]
We've added full Android 12 support in the Android rewrite we've been working on are preparing to ship: https://github.com/DoubleSymmetry/react-native-track-player/discussions/1264

**Comment by bradleyflood** [comment]
This should be resolved in #1371 

Can you help us test this out? @lelukas @annaostapenko22 @mrigo @ridik1803 @espenjanson @mayhizi 

Please and thank you all :)
